.github/workflows/node.js.yml

```diff
@@ -16,8 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        # TODO: hoist to 10.x when we bump the version supported in #5931
-        node-version: [ 8.x ]
+        node-version: [ 10.x, 14.x ]
```

Comment:
Should we not keep 8.x here until ">=8.0.0" in package.json changes?

Reply:
This has already happened for the base branch (`master-scoped-package`), but not `master`: https://github.com/mdn/browser-compat-data/blob/3331bed505ca9115150079eaa27a2d27d49dc301/package.json#L11 In other words, we'll still have Node 8.x on `master` until we're done with #6640.

Reply:
Oh, I didn't notice the target branch.